src/palace/manager/api/odl/api.py

```diff
@@ -745,6 +744,8 @@ def _update_hold_end_date(
                 days=default_reservation_period
             )
 
+        _db.expire_all()
```

Comment:
At least one unit test in the test_api.py tests was breaking because the holds collection associated with a patron was stale.  This update fixed it.  Surprisingly expiring the pool alone did not fix it.